src/main/java/org/opensearch/performanceanalyzer/commons/metrics_generator/linux/LinuxCPUPagingActivityGenerator.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.performanceanalyzer.commons.metrics_generator.linux;
+
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import org.opensearch.performanceanalyzer.commons.metrics_generator.CPUPagingActivityGenerator;
+import org.opensearch.performanceanalyzer.commons.os.ThreadCPU;
+
+public class LinuxCPUPagingActivityGenerator implements CPUPagingActivityGenerator {
+
+    private Map<String, Double> cpu;
+    private Map<String, Double[]> pagingActivities;
```

Comment:
Can this made part of the interface `CPUPagingActivityGenerator` ? A more appropriate name for `CPUPagingActivityGenerator` is `CPUPagingActivityStore`.